initfiles/sbin/complete-uninstall.sh.in

```diff
@@ -16,7 +16,14 @@
 #    along with All rights reserved. This program is free software: you can redistribute program.  If not, see <http://www.gnu.org/licenses/>.
 #############################################
 
-###<REPLACE>###
+INSTALL_DIR=/opt/HPCCSystems
+CONFIG_DIR=/etc/HPCCSystems
+ENV_XML_FILE=environment.xml
+ENV_CONF_FILE=environment.conf
+PID_DIR=/var/run/HPCCSystems
+LOCK_DIR=/var/lock/HPCCSystems
+LOG_DIR=/var/log/HPCCSystems
+INIT_PATH=/etc/init.d
```

Comment:
I think these need to be removed and just have the ###<REPLACE>### like the original, as that string gets 'replaced' by or expanded to the all these actual paths when installing.

Reply:
Got you. I will make the change.